src/View/ImageLoader.py

```diff
@@ -166,8 +166,11 @@ def load(self, interrupt_flag, progress_callback):
 
                     return True
         else:
-            self.load_temp_rtss(path, progress_callback, interrupt_flag)
-
+            try:
+                self.load_temp_rtss(path, progress_callback, interrupt_flag)
+            except ValueError as e:
+                raise ImageLoading.NotAllowedClassError(e)
```

Comment:
is the ValueError from load_temp_rtss due to an object not being in the allowed list of SOP Classes?
If so, great.
If the error might be something else... then that's not the appropriate error class to use.

Reply:
There is an error thrown (besides the newly created one) that occurs in this workflow. 

It also occurs in `src/Model/ROI.py` when StudyInstanceUID is not present.